### PR TITLE
Update operating-metrics.md

### PR DESCRIPTION
**Pull request by mollyyoung**
add operating metrics

### CODE REVIEW

**Comment by mollyyoung** [comment]
@comharris  I added the definitions here but they could probably be reorganized in terms of the headers :)

**Comment by comharris** [comment]
> @comharris I added the definitions here but they could probably be reorganized in terms of the headers :)

Thanks, @mollyyoung !

**Review by comharris — APPROVED** [review]

**Comment by comharris** [comment]
@mollyyoung Was there a doc to link in this section? (we can add the link, esp if it's a Google doc that has limited read access):

> ### MQL
Marketing Qualified Lead (MQL) is a lead that has reached a lead scoring threshold (100 points). Lead scoring model is based on firmographic, demographic, and behavioral information and can be seen here. (internal)(doc)

**Comment by it33** [comment]
Great stuff!

Is Alexis signed off on these definitions? Specifically SMB/Commercial/Enterprise definitions, also curious in general.

**Comment by mollyyoung** [comment]
Hello to you both,

The definitions were approved on 01/10 in a meeting with Alexis, Rich, Steve, and myself. 

Thank you, 
Molly

**Comment by mollyyoung** [comment]
I meant to attach the doc for lead scoring and will add tomorrow and cc you.

**Comment by mollyyoung** [comment]
@comharris @it33 I added the lead scoring doc (viewable to only those at Mattermost) and re-confirmed definitions with Alexis.  We should be good to merge pending approval from @it33.

**Comment by it33** [comment]
Okay from me. I think later we should figure out the approval process for metrics page.

File: source/guides/operating-metrics.md [hunk 1]
```diff
@@ -47,6 +47,28 @@ Decrease in [USD Cash Balance](#usd-cash-balance) from the first of the month to
...
+Sales Qualified Lead (SQL) is an opportunity created in Stage 1 or greater as either an SAO or Sales created opportunity that meets the pipeline requirements.
+
+### SAO
+Sales Accepted Opportunity (SAO) is an opportunity in stage 1 or greater. Created by a sales team member or by an SDR and accepted by a sales team member following an initial qualifying meeting (IQM) or SDR handoff. 
+
+### IQM
+Initial Qualifying Meeting (IQM) is a meeting scheduled by an SDR for a sales team member with a qualified lead. 
+
+### SMB 
+Companies less than 500 employees
```
Comment (it33) [inline source/guides/operating-metrics.md]: Should the upper end be 499 to be consistent with definitions below?

Comment (it33) [reply source/guides/operating-metrics.md]: @mollyyoung sorry, I was reading it again and would like to propose definition to make things extra clear--what is a company with 500 employees, and 5000 employees

**Comment by mollyyoung** [comment]
@it33 Thank you, that does provide more clarity.

**Pull request merged**